<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img_continue">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>325</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="grab">
       <property name="text">
        <string>grab</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="img_1">
       <property name="text">
        <string>img_1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="zwei">
       <property name="text">
        <string>zwei</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="cam_richt">
       <property name="text">
        <string>cam_richt</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="cam_link">
       <property name="text">
        <string>cam_link</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="construct">
       <property name="text">
        <string>construct</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="func_test">
       <property name="text">
        <string>func_test</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="sys_cal">
       <property name="text">
        <string>sys_cal</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="save_cal_param">
       <property name="text">
        <string>save_cal_param</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="undistort">
       <property name="text">
        <string>undistort</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="read_cal_param">
       <property name="text">
        <string>read_cal_param</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>0</y>
      <width>131</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="3" column="1">
      <widget class="QPushButton" name="set_param">
       <property name="text">
        <string>set_param</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="exposure_time">
       <property name="text">
        <string>exposure_time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="exposure_time_set"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="read_param">
       <property name="text">
        <string>read_param</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>8777219</height>
        </size>
       </property>
       <property name="text">
        <string>img name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="img_name"/>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="for_cal">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>for_cal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
